src/ontology/cl-edit.owl

```diff
@@ -22464,6 +22466,24 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0019019 "smooth muscle cell
 AnnotationAssertion(rdfs:label obo:CL_0019019 "tracheobronchial smooth muscle cell"@en)
 EquivalentClasses(obo:CL_0019019 ObjectIntersectionOf(obo:CL_0000192 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0007196)))
 
+# Class: obo:CL_0019031 (intestine goblet cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:1996606") Annotation(oboInOwl:hasDbXref "PMID:24942678") obo:IAO_0000115 obo:CL_0019031 "Goblet cells reside throughout the length of the small and large intestine and are responsible for the production and maintenance of the protective mucus blanket by synthesizing and secreting high-molecular-weight glycoproteins known as mucins. The intestinal goblet cells secrete the MUC2 mucin, as well as a number of typical mucus components: CLCA1, FCGBP, AGR2, ZG16, and TFF3.")
```

Comment:
Mention the species for which the markers are applicable. e.g.   In humans the intestinal goblet cells secrete...

Reply:
Thanks, done in 18cecee0f0c2b421ddadd33034c271ab1a551bb8